Corzbank.Helpers/MappingProfile.cs

```diff
@@ -12,10 +12,12 @@ public class MappingProfile: Profile
         public MappingProfile()
         {
             CreateMap<UserModel, User>()
-                .ForMember(u => u.UserName, opt=>opt.MapFrom(x=>string.Join('0', x.Firstname, x.Lastname)));
+                .ForMember(u => u.UserName, opt=>opt.MapFrom(um=>string.Join('0', um.Firstname, um.Lastname)));
```

Comment:
um**[space]**=>**[space]** 

User**n**ame [name in lowercase]

Reply:
UserName from IdentityUser